common/autoconf/flags.m4

```diff
@@ -1222,7 +1222,7 @@ AC_DEFUN([FLAGS_SETUP_COMPILER_FLAGS_FOR_JDK_HELPER],
 
   # Customize LDFLAGS for executables
 
-  $2LDFLAGS_JDKEXE="${$2LDFLAGS_JDK}"
+  $2LDFLAGS_JDKEXE="${$2LDFLAGS_JDKEXE} ${$2LDFLAGS_JDK}"
```

Comment:
Can this not be achieved without modifying openjdk source?

Reply:
thanks @keithc-ca. i moved the changes into `custom-spec.gmk.in`.